plugins/modules/remote_management/ipmi/ipmi_power.py

```diff
@@ -70,6 +92,28 @@
     returned: success
```

Comment:
This is no longer returned if `machine` is provided.
```suggestion
    returned: success and I(machine) is not provided
```